beacon-chain/powchain/service.go

```diff
@@ -545,6 +547,12 @@ func (s *Service) waitForConnection() {
 				}).Info("Connected to eth1 proof-of-work chain")
 				return
 			}
+			_, err := s.cfg.BeaconDB.GenesisState(s.ctx)
+			if err != nil {
```

Comment:
An error here doesn't imply genesis state is not available

Reply:
Fixed. Wasn't sure if we wanted to log a warning or an error here so kept it at warning for now. Lmk if you think this is Error worthy

/healthz returns:
```
*p2p.Service: OK
*powchain.Service: ERROR, genesis state missing, eth1 node out of sync
*attestations.Service: OK
*blockchain.Service: OK
*initialsync.Service: OK
*sync.Service: OK
*rpc.Service: ERROR, syncing
*gateway.Gateway: OK
*prometheus.Service: OK
```

Logs show:

```
[2021-07-02 07:44:19]  WARN powchain: Eth1 node is out of sync and genesis state is missing
[2021-07-02 07:44:24]  INFO initial-sync: Waiting for enough suitable peers before syncing required=3 suitable=2
[2021-07-02 07:44:30]  INFO initial-sync: Processing block batch of size 31 starting from  0xae789b9e... 20129/1628021 - estimated time remaining 288h9m9s blocksPerSecond=1.6 peers=5
[2021-07-02 07:44:34]  WARN powchain: Eth1 node is out of sync and genesis state is missing
^C[2021-07-02 07:44:41]  INFO node: Got interrupt, shutting down...
[2021-07-02 07:44:41]  INFO node: Stopping beacon node
[2021-07-02 07:44:41]  WARN initial-sync: Batch is not processed error=context canceled
```